core/src/player.rs

```diff
@@ -1156,6 +1165,12 @@ impl Player {
                 }
             });
         }
+
+        if let PlayerEvent::MouseLeave = event {
```

Comment:
Is there a reason why this didn't get a "MouseEnter" counterpart for symmetry?

Reply:
Sorry, late reply. `MouseLeave` was already present in `events.rs` and there's indeed no `MouseEnter`. The reason, [according to this answer](https://github.com/ruffle-rs/ruffle/pull/6006#discussion_r780883658), is that AS3 only has an [`Event.MOUSE_LEAVE`](https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/events/Event.html#MOUSE_LEAVE).

Reply:
Ah, I see. Strange!